sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -380,6 +394,34 @@ public void run() {
         });
     }
 
+    private synchronized void instanceSignOut() {
+        if (mTokenStorage.getToken() == null) {
+            AppCenterLog.warn( LOG_TAG, "Couldn't sign-out: authToken doesn't exist.");
+            return;
+        }
+        mSignInDelayed = false;
```

Comment:
Those 3 lines 402-404 should be shared with disabled code as disables does a sign out as part of disabling the module.

Reply:
@guperrot Do we need to remove an account on disabling the module?

Reply:
signOut is a sub-set of disabling